### DISCUSSION TITLE
the work flow dose not work proberly.

### DISCUSSION THREAD

**Discussion by mo-fouad** [DevSkillsHQ/discusssions#50] [Help] [answered]
I have done those steps as illustrated 
 
1- Check out [this list](https://help.alvalabs.io/en/articles/7972852-supported-coding-test-boilerplates) to pick a desired boilerplate and copy its name (e.g., mobile-boilerplate-flutter).
2- Go to the "Actions" tab of your GitHub repository and select the "Setup boilerplate" workflow in the left side panel.
3- In the "Run workflow" dropdown, paste the previously copied boilerplate name along with the branch name where you want the boilerplate to be imported (e.g., implementation) and click the "Run workflow" button.
4- After the workflow has finished, your selected boilerplate will be imported to the specified branch, and you can continue with your task there.
 
 and After the workflow is done, I found it only copied the app.json 
 I tested it 2 times with 2 different branches, but the same issues, 
 
 it should copy all the boilerplate, right?
 
 
![Screenshot 2023-11-13 at 15 46 53](https://github.com/DevSkillsHQ/discusssions/assets/24290379/63bb7f1a-2a3b-41d9-99df-21c8e961ae8d)

**Comment by fenske** [comment, marked as answer]
Hi @mo-fouad. There was an issue with the import script, which I now fixed.

I've imported the react-native boilerplate for you [here](https://github.com/ffc1e12/transaction-management-mobile-level-2_c2e1d27-1a9dg0/tree/implementation).

Sorry for the inconvenience, and I wish you the best of luck with the rest of your interview process!

BR,
Anton.

**Comment by mo-fouad** [comment]
@fenske  Thanks a lot for your answer, 
I would suggest that ou can update the boilerplate as well to the latest ExpoSDK 49, and React Native 0.72 this will make things a bit faster I guess, and more easer for the developers to focus only on the tasks. unless of course it is part of the task ^^ 

Best of luck :)

**Reply by fenske** [reply]
Yeah, that makes sense. Thx for the hint!
